### DISCUSSION TITLE
AlDente 1.29

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1381] [General]
### All
- The new M4 series MacBook Pros are now officially supported.
- Updated Ukrainian and French translations.
- Error messages now appear when using Top Up or Discharge while the device is unplugged.
- Fixed a bug that caused the popover to overlap with the dock in some configurations.
- Many smaller improvements and fixes.

### Pro
- A new "Power Adapter Specs" section was added to the Sensor Data Panel.
- A new "Apps Using Significant Energy" panel was added to the Stats Section.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.29'>AlDente 1.29</a>.</em>

**Comment by nthekim** [comment]
It seems that with the 1.29 new bug with charging exists! Despite being limited to 80%, the battery is charged to 88% after 10 hours. Seems to be micro charges...

**Reply by MatthiasKerbl** [reply]
Hi @nthekim,

I am sorry to hear you are having issues with AlDente. If possible, please share a debug file the next time this happens so that we can assess this issue better. You can find a guide on [How to generate and share a debug file](https://apphousekitchen.com/how-to-generate-and-share-a-debug-file/) on our blog.

Best,
Matthias
